### PR TITLE
PJ_SIM_SUNBNB_2023-11 [FE] Home Page Components Banner

### PR DESCRIPTION
**Pull request by andreenriquez11**
# Issue Link
https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-11

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1690527477852029

# Definition of Done
* [x] Create a component for home page for user to search listings
    * [x] SunBnB Description
    * [x] Search Component
        * [ ] Autocomplete search field
    * [x] Check In Date Input
    * [x] Check Out Date Input
    * [x] Search Button
    * [x] Divider Line

# Notes
- Installed vue-date-picker (npm install @vuepic/vue-datepicker)
- Please execute npm install again to reflect changes

# Pre-conditions
- User cannot input end date without inputting start date first
- User end date option is after the inputted start date
- Start date automatically sets to 2pm and end date sets to 12pm

# Expected Output
image

# Screenshots/Recordings
image

image

image

image

### CODE REVIEW

File: fe/src/components/DatePicker.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+<script>
+import VueDatePicker from "@vuepic/vue-datepicker";
+import "@vuepic/vue-datepicker/dist/main.css";
+
+export default {
+  components: { VueDatePicker },
+  data() {
+    return {
+      startDate: null,
```
Comment (LennardUy104) [inline fe/src/components/DatePicker.vue:32]: Set This as string ('')

File: fe/src/components/DatePicker.vue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        v-model="startDate"
+        :min-date="getNextDay(currentDate)"
+        :max-date="endDate || null"
+        placeholder="Start Date"
+        @input="setStartDateTo2PM"
+      />
+      <VueDatePicker
+        class="date-picker"
+        v-model="endDate"
+        :min-date="startDate ? getNextDay(startDate) : currentDate"
```
Comment (LennardUy104) [inline fe/src/components/DatePicker.vue:15]: Current date is not allowed
image

**Review by LennardUy104 — CHANGES_REQUESTED** [review]
Is it possible to make the startdate and enddate format similar to currentdate?
image

for database purposes

File: fe/src/components/DatePicker.vue (hunk 2 above)
Comment (andreenriquez11) [reply fe/src/components/DatePicker.vue:15]: Will do.

**Review by andreenriquez11 — COMMENTED** [review]
@LennardUy104 Resolved current date not allowed on startDate issue, and changed the format of startDate and endDate to the same of currentDate.

image

**Review by LennardUy104 — APPROVED** [review]
lgtm, great job
thank!

**Review by ejanton-sun — APPROVED** [review]
# Approve

**Pull request merged**